js_modules/dagster-ui/packages/ui-core/src/instance/backfill/BackfillActionsMenu.tsx

```diff
@@ -24,17 +24,11 @@ import {getBackfillPath} from '../../runs/RunsFeedUtils';
 import {runsPathWithFilters} from '../../runs/RunsFilterInput';
 import {AnchorButton} from '../../ui/AnchorButton';
 
-export function backfillCanCancelSubmission(backfill: {
+export function backfillCanCancel(backfill: {
   hasCancelPermission: boolean;
-  isAssetBackfill: boolean;
   status: BulkActionStatus;
-  numCancelable: number;
 }) {
-  return (
-    backfill.hasCancelPermission &&
-    ((backfill.isAssetBackfill && backfill.status === BulkActionStatus.REQUESTED) ||
-      backfill.numCancelable > 0)
-  );
+  return backfill.hasCancelPermission && backfill.status === BulkActionStatus.REQUESTED;
```

Comment:
since job backfills can be canceled via the daemon now, we don't need this additional check. I think this is vestigial from a time when job-backfills were canceled by terminating runs that were in-progress